pkg/csi/service/common/constants.go

```diff
@@ -431,6 +431,8 @@ const (
 	WorkloadDomainIsolationFSS = "workload-domain-isolation"
 	// VPCCapabilitySupervisor is a supervisor capability indicating if VPC FSS is enabled
 	VPCCapabilitySupervisor = "VPC_Supported"
+	// TODO: WCP_VMService_BYOK_FSS enables Bring Your Own Key (BYOK) capabilities.
+	BYOK_FSS = "byok"
```

Comment:
it is better to manage this with wcp capability so both vm service and csi can make use of common source of truth to determine if fss needs to be enabled or disabled.

Reply:
This was agreed not to be a capability but FSS in CSI